### PR TITLE
New release web page pointing to nightlies

### PR DESCRIPTION
**Pull request by PaulFreund**
Here is how it will look like, I still need to figure out integration:

image

Feedback welcome!

### CODE REVIEW

**Comment by jamiefaye** [comment]
It might be a good idea to put a link to the "official Synthstrom release build" firmware file(s) just so somebody who changes their mind can get ahold of the "antidote" if need be.

**Comment by PaulFreund** [comment]
I updated the page to include more notes about the official firmware. We can continually improve the page especially after we discussed more complex scenarios and dynamic data. I expect it to just work but we might need to fix the workflow after the PR if something unexpectedly does not.

image

File: .github/workflows/build-pages.yml [hunk 1]
```diff
@@ -4,12 +4,9 @@
...
 on:
-  # Automatically run after the build workflow completes, since that workflow
-  # must *always* run to get PRs approved.
-  workflow_run:
-    workflows: [Build]
-    types:
-      - completed
+  # Runs on pushes targeting the default branch
+  push:
+    branches: ["community"]
```
Comment (bobtwinkles) [inline .github/workflows/build-pages.yml:9]: This should have a `paths:` filter so the workflow only runs on commits that touch the `pages/` directory I think

**Review by bobtwinkles — APPROVED** [review]
Minor nit (I doubt the build will take enough time for it to matter) but otherwise LGTM. Thanks for carrying the torch forward on this one!

File: .github/workflows/build-pages.yml (hunk 1 above)
Comment (PaulFreund) [reply .github/workflows/build-pages.yml:9]: Fixed :) Thanks!

**Pull request merged**